package.json

```
@@ -467,7 +467,7 @@
         "test": "./scripts/test.sh",
         "package": "vsce package",
         "update-snapshot": "./scripts/test.sh -u",
-        "vscode:prepublish": "./scripts/build.sh",
+        "vscode:prepublish": "./scripts/build.sh && node esbuild.js",
```

shouldn't this use the `--production` flag? Does this minify the bundle even further?

```suggestion
        "vscode:prepublish": "./scripts/build.sh && node esbuild.js --production",
```